include/libint2/boys.h

```diff
@@ -1329,176 +1479,180 @@ struct TennoGmEval {
     } else
       mmin_interp = Exp ? -1 : 0;
 
-    // now compute the rest
-    for (long m = mmin_interp; m <= mmax; ++m) {
-      const Real* c_tuint =
-          c_ + (ORDERp1) * (ORDERp1) *
-                   (interval * (mmax_ - mmin_ + 1) +
-                    (m - mmin_));  // ptr to the interpolation data for m=mmin
+    try {
```

Comment:
why need exception handling here? should not use exception handling for non-exceptional control flow.